### ISSUE TITLE
Database Table Warning - Ajax

### ISSUE THREAD

**Issue by rvasquez151** [snipe/snipe-it#1347]
I installed the seeded data.  I did not know that it was going to install tons of data, so I deleted the entries from the tables. All is working well, except when I go to Asset Maintenance I get the Database Table Warning - Ajax.  I checked the assigned_to tables... empty.  Any suggestions?

**Comment by snipe** [comment]
Can you delete the data from the maintenances table?

**Comment by snipe** [comment]
In fact, if you have no production data in there yet, delete everything from `asset_logs` too.

**Comment by rvasquez151** [comment]
I had deleted everything from asset_logs table.  Didn't think about deleting seed data from maintenance table... that done it! Thank you once again!

**Comment by snipe** [comment]
Great!
